### DISCUSSION TITLE
AutoUpload

### DISCUSSION THREAD

**Discussion by cnagytcorp** [dzhuang/django-galleryfield#60] [Q&A] [answered]
Love your work but I can't figure out how to enable AutoUpload other than changing the line 2 in the defaults.py file...

**Comment by dzhuang** [comment, marked as answer]
You can override that in the `DJANGO_GALLERY_FIELD_CONFIG` settings, in your settings file. See the `jquery_file_upload_ui_options` in  [DJANGO_GALLERY_FIELD_CONFIG](https://django-galleryfield.readthedocs.io/en/latest/settings.html#django-gallery-field-config).

Simply put 

```python
DJANGO_GALLERY_FIELD_CONFIG = {
    "jquery_file_upload_ui_options": {
        "autoUpload": True,
    },
}

```

will suffice.
